cmd/create.go

```diff
@@ -93,22 +93,30 @@ A full list of component types that can be deployed is available using: 'odo com
 			os.Exit(1)
 		}
 
+		fmt.Printf("Creating component %s from build type %s\n", componentName, componentType)
```

Comment:
How about "Creating %s component called %s" ?
So it the resulting string would be something like this:

Creating php component called frontend.
or
Creating nodejs component called nodejs.